src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -8905,9 +8907,9 @@ static void Main()
             Assert.True(m3Item8.GetAttributes().IsEmpty);
             Assert.Null(m3Item8.GetUseSiteDiagnostic());
             Assert.False(m3Item8.Locations.IsDefaultOrEmpty);
-            Assert.Equal("int", m3Item8.DeclaringSyntaxReferences.Single().GetSyntax().ToString());
```

Comment:
This is example where things went wrong. 
"int" is not a DeclaringSyntaxReference for "Item8". It is a Location associated with the symbol, but not its declaring syntax. In fact Item8 was implicitly declared by the compiler. User did not declare it directly at all.

 If IDE tries renaming "int" in response to editing of "Item8" at a use site, bad things will happen.